test/integration/policy_imagemanifest_test.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+const policyIMVURL = "https://raw.githubusercontent.com/stolostron/policy-collection/main/stable/SI-System-and-Information-Integrity/policy-imagemanifestvuln.yaml"
```

Comment:
```suggestion
const policyIMVURL = policyCollectSIURL + "policy-imagemanifestvuln.yaml"
```

Reply:
Can you also move this block inside `Describe`? We want to reduce the package level variables by scope it to function.